#source/scss/style.scss

```diff
@@ -0,0 +1,98 @@
+@import "nullstyle.scss";
+
+
+ body{
+ 	margin: 0 auto;
+ 	font-family: 'Montserrat', sans-serif;
+ }
+ @import "../fonts/font.css";
+
+ // ======================================= header
+.header {
+	padding: 24px 0 0 0; 
+}
+.header__container {
+	display: flex; 
+	justify-content: space-between;
+	align-items: center;
+}
+.header__logo {
+	padding: 0 0 0 0;
+}
+.logo {}
+.header__menu {
+	width: 490px;
+}
+.menu__row{
+	display: flex;
+	justify-content: space-between;
+}
+.menu__item {
+	font-weight: 600;
+	font-size: 15px;
+	line-height: 18px;
+	color: #000;
+	cursor: pointer;
+	&:hover{
+		color: #E51937;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5633737)